cobc/ChangeLog

```diff
@@ -190,6 +190,12 @@
 	* typeck.c (cb_build_move_literal): optimized output for literals to
 	  fields with BLANK WHEN ZERO and SIGN LEADING
 
+
```

Comment:
meh, that was me (there's now an empty line too much)

Reply:
Ok, I removed this line :)